library/cc/response_headers.cc

```diff
@@ -0,0 +1,21 @@
+#include "response_headers.h"
+
+// NOLINT(namespace-envoy)
+
+int ResponseHeaders::http_status() const {
+  if (!this->contains(":status")) {
+    throw std::out_of_range("ResponseHeaders does not contain :status");
```

Comment:
`out_of_range` doesn't feel right when the function doesn't take an arg. maybe change to something that explicates this is an internal issue